internal/pkg/cli/flag.go

```diff
@@ -18,7 +18,7 @@ const (
 	nameFlag    = "name"
 	svcFlag     = "svc"
 	envFlag     = "env"
-	appTypeFlag = "app-type"
+	svcTypeFlag = "svc-type"
```

Comment:
Maybe we should add flag aliases for service as well.

Reply:
We already have short flag name for service. Do you mean service type?